### PR TITLE
Docs: Added documentation for the assert argument

### PR DESCRIPTION
**Pull request by blacksun1**
Fixes: #634

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -447,6 +447,58 @@ If you would like to run a different linter, or even a custom version of eslint
 pass the `-n` or `--linter` argument with the path to the lint runner.  For example,
 if you plan to use jslint, you can install `lab-jslint` then pass `--linter node_modules/lab-jslint`.
 
+## Integration with an assertion library
+
+Using the `--assert` argument allows you to integrate Lab with your favourite assertion library. It works by
```
Comment (geek) [inline README.md]: favourite -> favorite

**Comment by geek** [comment]
@blacksun1 thanks, it looks good, just the minor nit

File: README.md (hunk 1 above)
Comment (blacksun1) [inline README.md]: Hey, I am an Aussie living in the UK so, sorry about that. You want me to change it?

Comment (geek) [inline README.md]: No problem, this is merged and updated!

Thanks for the help.

Comment (geek) [inline README.md]: Also, I hope I never have to play you in scrabble, seems like an unfair advantage with all of the extra letters :)

**Pull request merged**